packages/bpk-component-calendar/src/BpkCalendarContainer.js

```diff
@@ -41,9 +41,18 @@ const focusedDateHasChanged = (currentProps, nextProps) => {
   const rawNextSelectedDate = nextProps.selectedDate || nextProps.date;
   const rawSelectedDate = currentProps.selectedDate || currentProps.date;
 
-  return (
-    rawNextSelectedDate && !isSameDay(rawNextSelectedDate, rawSelectedDate)
-  );
+  if (!rawSelectedDate && !rawNextSelectedDate) {
+    return false;
+  }
+
+  if (
+    (rawSelectedDate && !rawNextSelectedDate) ||
+    (!rawSelectedDate && rawNextSelectedDate)
```

Comment:
This could be simplified, but maybe you were deliberately trying to make it explicit?
```suggestion
    !rawSelectedDate || !rawNextSelectedDate
```

Reply:
Going for deliberately explicit to:
- make it easy for future readers to scan and know the intent is 'prev is null and next is not OR prev is null/undefined and next is not'
- make it as easy as possible for reviewers to spot mistakes in my intent, as this is the first time I have looked at this component so may have missed context myself